main.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <string.h>
+
+
+
+
+
+void swap(char* a1, char* a2)
```

Comment:
Change the name of the function to swapElementsOfString. This name has only one meaning, by comparison, the previous one can refer to many things